dist/stat_data.py

```diff
@@ -274,6 +274,7 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_lookaside_cursor_wait_internal', 'cache overflow cursor internal thread wait time (usecs)'),
     CacheStat('cache_lookaside_entries', 'cache overflow table entries', 'no_clear,no_scale'),
     CacheStat('cache_lookaside_insert', 'cache overflow table insert calls'),
+    CacheStat('cache_lookaside_ondisk', 'the ondisk size of the cache overflow table'),
```

Comment:
We want to keep the earlier text similar so it appears in t2 and such together with other similar items. I suggest changing the description to `cache overflow table on-disk size`.

Reply:
1573b8f

Ah, so it gets sorted by name then by description.
Thanks, I'll note this in the doc also.